modules/nw-ovn-k-anp-best-practices.adoc

```diff
@@ -0,0 +1,58 @@
+
+* When using `0.0.0.0/0` or `::/0` to create a strong `Deny` policy, it is recommended that you check to make sure that you have higher priority `Allow` rules for essential traffic.
+
+* `Pass` is a special action used to delegate policy decision to the `NetworkPolicy` layer. You might consider using this action when considering the interoperability between ANP and your `NetworkPolicy`.
+
+* Selectors across multiple rules should not overlap. This ensures that that the same IPs do not appear in multiple policies, which can cause performance and scale limitations due to an OVN-Kubernetes limitation.
+
+* Using `namedPorts` is not performance efficient since each name has to get evaluated against the `subject` of the policy and `peer` pod's container ports. You might use this with caution in larger clusters where you have many pods and each pod has many containers.
+
+[id="considerations-using-banp_{context}"]
+== Considerations for using BaselineAdminNetworkPolicy
+
+* There is only one `BaselineAdminNetworkPolicy` (BANP) for the cluster be careful in defining it. The following are supported uses for BANP that administrators might consider in designing their BANP:
+
+** You can set a default deny policy for ingress in user namespaces. This BANP will make it so that users have to add `NetworkPolicy` to allow the ingress traffic that they want to allow, and if they do not add network policies for ingress it will be denied.
+
+** You can set a default deny policy for egress in user namespaces. This BANP will make it so that users have to add `NetworkPolicy` to allow the egress traffic that they want to allow, and if they do not add network policies it will be denied.
+
+** You can set a default allow policy for egress to DNS.This BANP will make it so that users in namespaces do not have to set an allow egress `NetworkPolicy` to DNS.
+
+** You can set an egress policy that allows egress to all pods and denies access to `0.0.0.0/0` and `::/0` in user namespaces. This BANP will make it so users still have to allow egress by default for any internal traffic in the cluster, but also have to deny by default any egress traffic in the cluster, and they have to add `NetworkPolicy` to allow the cluster egress traffic that they want to allow.
+
+* Be careful that you scope your BANP so that it only denies traffic in user namespaces and not to system namespaces. This is because the system namespaces do not have `NetworkPolicy` to override your BANP.
+
+[id="differences-with-netpol_{context}"]
+= Differences to consider between NetworkPolicy and ANP
+
+* Use workload specific labels to secure your workloads and then use those labels as the namespace selector instead of using `{}` string.
```

Comment:
```suggestion
* Use explicit labels to reference your workloads within ANP and BANP rather than using the empty (`{}`) catch all selector to avoid accidental traffic selection.
```